0001-Enable-procompiled-headers.patch

```diff
@@ -0,0 +1,448 @@
+From e3915a9be0e4d90476264ab8245a3ac02a6d4b4b Mon Sep 17 00:00:00 2001
```

Comment:
I don't understand why do we need this patch-file here.

Reply:
Removed